src/main/java/org/folio/service/transactions/AllOrNothingTransactionService.java

```diff
@@ -97,20 +100,18 @@ private Future<Void> cleanupTempTransactions(Transaction transaction, DBClient c
    *
    * @return future with void
    */
-  private Future<Void> processAllOrNothing(Transaction transaction, DBClient client, BiFunction<List<Transaction>, DBClient, Future<Void>> operation) {
-    return transactionSummaryService.getAndCheckTransactionSummary(transaction, client)
-      .compose(summary -> addTempTransactionSequentially(transaction, client)
+  private Future<Void> processAllOrNothing(Transaction transaction, DBConn conn, BiFunction<List<Transaction>, DBConn, Future<Void>> operation) {
+    return transactionSummaryService.getAndCheckTransactionSummary(transaction, conn)
+      .compose(summary -> addTempTransactionSequentially(transaction, conn)
         .compose(transactions -> {
           if (transactions.size() == transactionSummaryService.getNumTransactions(summary)) {
```

Comment:
**Minor**
Not the main goal of this ticket although it will make sense to simplify this by using pattern:
```
if (some condition) {
  return Future.succeededFuture();
}
main logic
```

Reply:
done